src/store/participantsStore.js

```diff
@@ -93,7 +93,9 @@ const mutations = {
 	 * @param {string} token the conversation to purge;
 	 */
 	purgeParticipantsStore(state, token) {
-		Vue.delete(state.participants, token)
+		if (state.participants[token]) {
```

Comment:
Maybe add a debug if that is not the case, so you can track weird stuff like that.
Theoretically it should never happen, but yu never know, it's always nice to have debugs liek that :)

Reply:
What do you mean here exactly?

Reply:
If the token given is not in the participants list (so in an else condition here), it's very unlikely to happen I guess, but having debug for those weird use cases can help you a lot if someone experience weird stuff :)

Reply:
so `console.debug('..')`?